### DISCUSSION TITLE
Nesting Lists

### DISCUSSION THREAD

**Discussion by Redninja106** [community/community#13377] [Other Feature Feedback, Questions, & Ideas]
It would be nice if lists were capable of containing other lists, making them essentially folders for our starred repos. This would be big help for those who have hundreds or even thousands of stars and want to keep them as organized as possible.

I’m aware this could complicate lists significantly, but I think its a feature that I and many others would love to see.

**Comment by barseghyanartur** [comment]
Yep. And once again, please add categorization to Gists too!

**Comment by AIRCOURIELtd** [comment]
> 
> It would be nice if lists were capable of containing other lists, making them essentially folders for our starred repos. This would be big help for those who have hundreds or even thousands of stars and want to keep them as organized as possible.
> 
> I’m aware this could complicate lists significantly, but I think its a feature that I and many others would love to see.

Yes
